composer/utils/checkpoint.py

```diff
@@ -165,13 +165,22 @@ def format(self, state: State, is_deepspeed: bool = False, keep_placeholders: bo
 def is_checkpoint_legacy_sharded(object_store: Optional[ObjectStore], source_path: str):
     metadata_path = str(Path(source_path) / Path('.metadata'))
     if object_store is None:
+        # If directory doesn't exist, then it's not any checkpoint.
+        if os.path.exists(source_path):
+            raise FileNotFoundError(f"Couldn't find the directory {source_path}")
```

Comment:
Just to be clear, because this is in `load_checkpoint`, it should not affect autoresume related logic, right?

Reply:
uh it shouldn't